spec/models/couch_spec.rb

```diff
@@ -15,4 +15,36 @@
     it { should have_many(:reservations).through(:nights) }
   end
 
+  context "methods" do
+    before do
+      create_list(:couch, 2, city: "This City")
+      create(:couch, city: "That City")
+    end
+
+    describe ".in_city()" do
+      it "returns couches for a given city" do
+        result = Couch.in_city("This City")
```

Comment:
@tmikeschu I think the city search should be case insensitive.  Would you mind adding spec/code for that additional capability?